third_party/terraform/tests/resource_active_directory_domain_update_test.go

```diff
@@ -0,0 +1,78 @@
+package google
+
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+)
+
+func TestAccActiveDirectoryDomain_update(t *testing.T) {
```

Comment:
I'm having trouble running the tests in CI. The initial POST request is returning a 403 with "Write access to project 'ci-test-project-188019' was denied". Have you seen this before? What do we need to do to get it working?

Reply:
role required is - roles/managedidentities.admin